### DISCUSSION TITLE
Add auto scaling and Open Lite Speed ​​support

### DISCUSSION THREAD

**Discussion by Deooz** [andreapollastri/cipi#446] [Ideas]
I don't know how complicated it would be to implement something like this, but it would be great if the panel could automate this, i.e. be able to deploy 20 1GB ram servers for 20 different sites that can handle traffic spikes. This is the first time I'm commenting on a repository I hope I'm not violating any rules. Regards

**Comment by andreapollastri** [comment]
Nope, nginx only!
